src/libraries/Common/tests/System/Net/EnterpriseTests/setup/apacheweb/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM httpd:2.4
```

Comment:
Could you create a docker image with all this already done? Azure container registry has the ability to automatically rebuild the container based on triggers such as the base image being updated. See https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview

Reply:
We're looking into caching some of these images to reduce image build time, especially for the 'linuxclient' container. However, once built on the local dev machine, it is cached by the docker system on the host machine.

It's always a trade-off with pre-building and storing these images since it creates another place to maintain infrastructure. We do have our existing Azure Container Registry, mcr.microsoft.com/dotnet-buildtools/prereqs, which might use for this purpose. This will be a workitem for a future PR after discussions with our infra group.